web-platform/website/src/components/pages/get-extension/GetExtensionView.tsx

```diff
@@ -72,20 +72,19 @@ export function GetExtensionView() {
       </Row>
       <Row className="mb-3">
         <Col>
-          <LoginButton className={PrimaryButton}>
-            {allStudies && (
-              <a
-                className={LinkStyles.NoUnderline}
-                href={browserType === BrowserType.Chrome ? chromeLink : fxLink}
-                target="_blank"
-                rel="noreferrer"
-                onClick={() => {
-                  router.push("/");
-                }}
-              >
-                {strings.getExt}
-              </a>
-            )}
+          <LoginButton
+            onClick={() => {
+              allStudies &&
```

Comment:
IMO this `allStudies` check should live outside the button; i.e. the button shouldn't appear at all until `allStudies` has loaded.

I think this can be accomplished with something like:
```
{allStudies && <LoginButton>...</LoginButton>}
```